cllc-public-app/Controllers/FederalTrackingController.cs

```diff
@@ -0,0 +1,128 @@
+using System.Collections.Generic;
+using System.IO;
+using CsvHelper;
+using Gov.Lclb.Cllb.Interfaces;
+using Gov.Lclb.Cllb.Interfaces.Models;
+using Gov.Lclb.Cllb.Public.ClassMaps;
+using Gov.Lclb.Cllb.Public.ViewModels;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Gov.Lclb.Cllb.Public.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
```

Comment:
What policy should I be adding to this Authorize annotation? I see "Business-User" is maybe the only one we have available? Would that be sufficient?

Reply:
If you want this to be only available to BCEID users then add Business-User.  

As it stands right now, the controller is available to both worker users and bceid users.